src/app/pages/class-details/class-details.view/class-details.view.component.html

```diff
@@ -7,16 +7,16 @@
         <div class="class-details">
           <div class="class-detail-field">
             <mat-form-field>
-              <input type="text" matInput [(ngModel)]="name" placeholder="שם הכיתה" (change)="detailChanged.emit({name: name, grade: grade})"
+              <input [attr.data-test-id]="'class-name-input'" type="text" matInput [(ngModel)]="name" placeholder="שם הכיתה" (change)="detailChanged.emit({name: name, grade: grade})"
                 required>
               <i class="material-icons" matSuffix>mode_edit</i>
             </mat-form-field>
           </div>
 
           <div class="class-detail-field">
             <mat-form-field class="class-detail-field">
-              <mat-select [(value)]="grade" placeholder="שכבה" (selectionChange)="detailChanged.emit({name: name, grade: grade})">
-                <mat-option *ngFor="let key of gradesKeys" [value]="key" selected="key === grade">
+              <mat-select [attr.data-test-id]="'grade-select'" [(value)]="grade" placeholder="שכבה" (selectionChange)="detailChanged.emit({name: name, grade: grade})">
```

Comment:
hey, is the binding needed? can we just use the `data-` attribute?

Reply:
Done.